alvr/server_core/src/tracking/mod.rs

```diff
@@ -373,6 +467,16 @@ pub fn tracking_loop(
                 &tracking.device_motions,
             );
 
+            if !tracking.markers.is_empty() {
+                tracking_manager_lock.report_markers(tracking.markers);
```

Comment:
We only use the trackers in the recenter function and that makes this quite weird.
Is this some provision to be able to extract other trackers later on?
Because otherwise I'd suggest just directly extract the settings for marker based reloc and do sth like
```rs
if marker_recenter_enabled {
  tracking_manager_lock.recenter_markers(marker_config, markers); 
}
```

Reply:
Yeah i see that markers could be used as aid in body tracking, that is 3dof + low/random frequency positional info. Integration with the slimevr server could come soon (convo in #dev)